omod/src/main/webapp/resources/scripts/jquery/autocomplete/OpenmrsAutoComplete.js

```diff
@@ -250,11 +250,8 @@ function CreateCallback(options) {
 		
 		textShown = imageText + textShown + ageText; // space was inserted into beginning of 'textShown' var
 
-		textShown = $j('<div/>').text(textShown).html()
```

Comment:
Is this intentionally reverting the change made here? https://github.com/openmrs/openmrs-module-legacyui/commit/fd7227cc412724dfe7dcb58bc37ad75921debfc5#diff-ba564bbe7692a2facf544e2b0de8450169b3467afee7aaf1f3b69c2e6e407725L247